### PR TITLE
Add AlmaLinux and Rocky

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: lib/puppet_metadata/operatingsystem.rb [hunk 1]
```diff
@@ -131,6 +131,15 @@ class OperatingSystem
         '5.04' => '2006-10-31',
         '4.10' => '2006-04-30',
       },
+      # https://almalinux.org/ FAQ at the bottom
+      'AlmaLinux' => {
```
Comment (ekohl) [inline lib/puppet_metadata/operatingsystem.rb:135]: I kept the operating systems sorted alphabetically. Can we keep that?

Comment (bastelfreak) [reply lib/puppet_metadata/operatingsystem.rb]: ah sure. done. Also added a comment that it's alphabetically

File: lib/puppet_metadata/operatingsystem.rb [hunk 2]
```diff
@@ -8,7 +8,13 @@ module PuppetMetadata
   class OperatingSystem
     # The EOL dates for the various operating systems
     # @see .eol_date
+    # please add new OSes sorted alphabetically
```
Comment (ekohl) [inline lib/puppet_metadata/operatingsystem.rb:11]: The yard documentation (`bundle exec yard && xdg-open doc/index.html`) renders in a poor way now:
image

I think what would work is:
```ruby
# please add new OSes sorted alphabetically

# The EOL dates for the various operating systems
# @see .eol_date
EOL_DATES = {
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: lib/puppet_metadata/operatingsystem.rb (hunk 2 above)
Comment (bastelfreak) [reply lib/puppet_metadata/operatingsystem.rb]: updated it. I don't think yard detects this as 'wrong' so we can't easily write a rake task that verifies this?

Comment (ekohl) [reply lib/puppet_metadata/operatingsystem.rb]: Nope. It's "just text" so you need some intelligence to determine it. Tools you can think about are diffing the entire docs, but that has so many other issues that I doubt that it's worth the effort.

**Review by ekohl — APPROVED** [review]

**Pull request merged**